src/codegen/bitfield_unit.rs

```diff
@@ -0,0 +1,84 @@
+#[repr(C)]
+#[derive(Copy, Clone, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub struct __BindgenBitfieldUnit<Storage, Align>
+where
+    Storage: AsRef<[u8]> + AsMut<[u8]>,
+{
+    storage: Storage,
+    align: [Align; 0],
+}
+
+impl<Storage, Align> __BindgenBitfieldUnit<Storage, Align>
+where
+    Storage: AsRef<[u8]> + AsMut<[u8]>,
+{
+    #[inline]
+    pub fn new(storage: Storage) -> Self {
+        use std::mem::align_of;
```

Comment:
This somehow needs to be `core` when `--use-core` is passed.

Reply:
@emilio The simplest solution is to remove the assertion (and its use of `align_of`) as it's not needed currently and it's very unlikely that anyone will trip over the assumption in the future.